test/es_test_case.py

```diff
@@ -40,6 +40,12 @@ def setUpClass(cls):
         except BaseException:  # no coverage
             cls._kill_containers()
             raise
+        else:
+            # By default `max_compilations_per_minute` is set to 15, we must increase that threshold in order to execute
+            # dynamic header row and HashCode generation for test_full_hash_validity.
+            # If this also happens in production code, we should move this code to be run as part of `make terraform`,
+            # as a provisioner, for example. Or we can precompile the Painless scripts as part of `make deploy`.
+            cls.es_client.cluster.put_settings(body=dict(persistent={'script.max_compilations_per_minute': 60}))
```

Comment:
Line is too long. Can we convert the `dict` call to a literal and wrap this a bit?